src/ZstdSharp/Unsafe/Bitstream.cs

```diff
@@ -10,7 +10,7 @@ namespace ZstdSharp.Unsafe
 {
     public static unsafe partial class Methods
     {
-#if NET8_0_OR_GREATER
+#if NET7_0_OR_GREATER
```

Comment:
As far as I know this optimization only works for `byte` / `sbyte` / `bool` in .NET 7 and extended in .NET 8
This part of code is made by [ZstdSharp.Generator](https://github.com/oleg-st/ZstdSharp.Generator)
See
https://github.com/oleg-st/ZstdSharp.Generator/blob/b7810ee1b65ef90b39a63caa3833b463bb69540d/src/ZstdSharp.Generator/CodeGenerator/CodeGenerator.Helper.cs#L743

Reply:
The required [runtime API](https://learn.microsoft.com/en-us/dotnet/api/system.runtime.compilerservices.runtimehelpers.createspan) was added in .NET 7, but compiler support was added in the .NET 8 SDK. I built the .NET 7 assembly locally with the .NET 9 SDK and it produced the optimized code.

![image](https://github.com/user-attachments/assets/0865594e-c774-49f3-8089-633888923e96)

And that's how the code for this library was generated, awesome!